src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -384,7 +388,13 @@ public override EnvDTE.FileCodeModel GetFileCodeModel(DocumentId documentId)
                 this.EnsureEditableDocuments(changedDocs);
             }
 
-            return base.TryApplyChanges(newSolution, progressTracker);
+            if (base.TryApplyChanges(newSolution, progressTracker))
+            {
+                NotifyRefactorChanges(changedDocs, currentSolution);
```

Comment:
Now that we're doing this, we probably need to delete the ISymbolRenamedCodeActionOperationFactoryWorkspaceService or we might actually double notify things. (For the record, I didn't know that type existed until now when I was trying to find something else.) Also, we may have to delete some of this code now too:

https://github.com/dotnet/roslyn/blob/4cd280b52e1c63f57c48d405fec32f5a8cfcf553/src/VisualStudio/Core/Impl/CodeModel/AbstractCodeModelService.cs#L498-L513

Note that code *also* dismisses rename tracking, which might be a wise thing to handle in NotifyRefactorChanges as well. @dpoeschl might have a nicer way to hook that up though -- I wonder if rename tracking should just have an IRefactorNotifyService exported to get notified of a global rename rather than the direct coupling.